[2022-12-26T07:43:52.704+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: bertopic_autopik_airflow.save_processed_data manual__2022-12-26T07:42:34.815881+00:00 [queued]>
[2022-12-26T07:43:52.718+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: bertopic_autopik_airflow.save_processed_data manual__2022-12-26T07:42:34.815881+00:00 [queued]>
[2022-12-26T07:43:52.718+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-12-26T07:43:52.719+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-12-26T07:43:52.719+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-12-26T07:43:52.740+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): save_processed_data> on 2022-12-26 07:42:34.815881+00:00
[2022-12-26T07:43:52.748+0000] {standard_task_runner.py:54} INFO - Started process 264 to run task
[2022-12-26T07:43:52.751+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bertopic_autopik_***', 'save_processed_data', 'manual__2022-12-26T07:42:34.815881+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/bertopic_autopik_***.py', '--cfg-path', '/tmp/tmpt_c1f8go']
[2022-12-26T07:43:52.752+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask save_processed_data
[2022-12-26T07:43:52.753+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/bertopic_autopik_***.py
[2022-12-26T07:44:06.918+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-12-26T07:44:07.068+0000] {task_command.py:384} INFO - Running <TaskInstance: bertopic_autopik_airflow.save_processed_data manual__2022-12-26T07:42:34.815881+00:00 [running]> on host c4b524c24ec1
[2022-12-26T07:44:07.224+0000] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=hojun.seo@pikurate.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bertopic_autopik_***
AIRFLOW_CTX_TASK_ID=save_processed_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-26T07:42:34.815881+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-26T07:42:34.815881+00:00
[2022-12-26T07:44:20.982+0000] {warnings.py:109} WARNING - /opt/***/dags/functions/data.py:123: DtypeWarning: Columns (0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,26,27,28) have mixed types. Specify dtype option on import or set low_memory=False.
  data = pd.read_csv(df)

[2022-12-26T07:44:41.671+0000] {logging_mixin.py:117} INFO - processed_data shape is:  (544291, 29)
[2022-12-26T07:44:41.870+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-26T07:44:41.887+0000] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=bertopic_autopik_***, task_id=save_processed_data, execution_date=20221226T074234, start_date=20221226T074352, end_date=20221226T074441
[2022-12-26T07:44:42.368+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-12-26T07:44:42.433+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
